docs/public/version.json

```diff
@@ -0,0 +1 @@
+{ "versions": ["next", "3.5", "3.4"] }
```

Comment:
Oh ok so you defined `version.json` in this PR... it's probably easier that these are one PR